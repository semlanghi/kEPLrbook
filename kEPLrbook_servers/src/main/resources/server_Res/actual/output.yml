---
inputs:
  500:
  - sum(id): 1
  - sum(id): null
  1000:
  - sum(id): null
