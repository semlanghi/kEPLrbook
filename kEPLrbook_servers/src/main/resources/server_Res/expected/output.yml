inputs:  
  500 :
    - type: "NbaGamer"
      id: 1
      awayTeam: "Sacramento Kings"
      homeTeam: "New York Knicks"
      
  1000 :
    - type: "NbaGamer"
      id: 2
      awayTeam: "Sacramento Kings"
      homeTeam: "New York Knicks"
      
    - type: "NbaGamer"
      id: 3
      awayTeam: "Sacramento Kings"
      homeTeam: "New York Knicks"